services/efs-guardian/src/simcore_service_efs_guardian/core/settings.py

```diff
@@ -61,7 +68,7 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         description="Linux group name that the EFS and Simcore linux users are part of"
     )
     EFS_DEFAULT_USER_SERVICE_SIZE_BYTES: ByteSize = Field(
-        default=parse_obj_as(ByteSize, "500GiB")
+        default=TypeAdapter(ByteSize).validate_python("500GiB")
```

Comment:
Q: I’ve been seeing this change everywhere. What is it about?

Reply:
parse_obj_as is retired ;)